yt_dlp/extractor/facebook.py

```diff
@@ -511,7 +532,7 @@ def extract_relay_prefetched_data(_filter):
 
         if not video_data:
             data = extract_relay_prefetched_data(
-                r'"(?:dash_manifest|playable_url(?:_quality_hd)?)"\s*:\s*"[^"]+"')
+                r'"(?:dash_manifest|playable_url(?:_quality_hd)?)')
```

Comment:
Is this necessary? I'm not saying it's wrong to change it, just wondering why

Reply:
Keeping that trailing pattern breaks at least 1 test actually: `https://www.facebook.com/video.php?v=274175099429670`
And I don't see a reason to keep it anyway. Dependent pattern in `extract_relay_data` makes sure we collect only valid json body of `script`s. This pattern just narrows it down a bit.